WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/DashboardGoogleDomainsCardView.swift

```diff
@@ -32,6 +41,7 @@ struct DashboardGoogleDomainsCardView: View {
         HStack {
             Button {
                 print("Transfer your domains tapped")
```

Comment:
Probably this print should be removed or refactored to `DDLogInfo` if you prefer to log that message.